### ISSUE TITLE
The following test.adoc produces incorrect docbook.xml

### ISSUE THREAD

**Issue by rmx-it** [asciidoctor/asciidoctor.org#371]
Hi,
since I use a lot of verbatim commandline examples in my .adoc documents I found some strange
behaviour of asciidoctor when producing docbook output.

````asciidoc
= Book Title
:Author: xxx yyy
:Email: <xxx@yyy.com>
:doctype: book
:numbered:
:toc: left
:toclevels: 3
:toc-title: Contents
:sectanchors:
:icons: font
:linkcss:
:experimental:
:source-language: shell
:leveloffset: 0

== Test1

* This
* Example
* would `/xxx/${XXX}/.xxx.*`
* work.

== Test2

* This Example does `/xxx/${XXX}/.xxx.*` not `/xxx/${XXX}/.xxx.*` work!
````
The chapter Test1 alone gives correct output and no xmllint error.
The example line in chapter Test2 gives the following error:

````shell
test.xml:21: parser error : Opening and ending tag mismatch: emphasis line 21 and literal
a>This Example does <literal>/xxx/${XXX}/.xxx.<emphasis role="strong"></literal>
                                                                               ^
test.xml:21: parser error : Opening and ending tag mismatch: literal line 21 and emphasis
xxx.<emphasis role="strong"></literal> not <literal>/xxx/${XXX}/.xxx.</emphasis>
                                                                               ^
````

This seems like a bug of asciidoctor to me.

**Comment by mojavelinux** [comment]
The problem you are running into is that we changed the behavior of backticks in Asciidoctor 1.5.0. They no longer passthrough the content. Instead, you need to add pluses inside the backticks to escape the content.

```
This Example does `+/xxx/${XXX}/.xxx.*+` not `+/xxx/${XXX}/.xxx.*+` work!
```

Another option is to set the `compat-mode` attribute on the document.

```
:compat-mode:
```

This is described in the migration guide.

http://asciidoctor.org/docs/migration/

I realize that without the passthrough, the asterisks are not only interpreted as bold, but placed in the wrong location. This can only really be solved by fixing the inline parsing to use a proper parser which we are now beginning to explore. The regular expression style parsing just gets this wrong and it's incredibly hard to fix with that approach.

**Comment by jaredmorgs** [comment]
I wonder if we should move http://asciidoctor.org/docs/migration/#migration-scenarios into the User Guide near those sections that describe monospace (http://asciidoctor.org/docs/user-manual/#mono)

Another questions is where is inline code using backticks explicitly described in the User Guide?

**Comment by mojavelinux** [comment]
I'm not sure about moving the migration into the User Guide. I don't like it when User Guides spend too much time talking about what's done and gone as it just confuses newcomers. I like the idea that we have a dedicated migration page. Of course, we could link to it at strategic locations. We could mention in an admonition in the monospace section that the meaning of backticks differs from AsciiDoc Python and link to the migration page.

The backticks are described in http://asciidoctor.org/docs/user-manual/#mono. However, that section should finish by explaining when to use pluses inside the backticks to prevent substitutions without the monospaced text.

**Comment by jaredmorgs** [comment]
Adding nodoc as this is part of the Support milestone.

**Comment by jaredmorgs** [comment]
>The backticks are described in http://asciidoctor.org/docs/user-manual/#mono. However, that section should finish by explaining when to use pluses inside the backticks to prevent substitutions without the monospaced text.

This appears to be the only action item in this issue. Do you agree @mojavelinux ?

**Comment by mojavelinux** [comment]
Correct. Let's get a separate issue going for that. The topic is "literal monospace", where literal means preventing interpolation and formatting. Then, we can close this issue.

Also, we are in the process of adding a migration section from AsciiDoc Python to the user guide. See #540.
